src/components/generator/GenerateDialog.js

```diff
@@ -0,0 +1,98 @@
+import Notifications from "../common/Notifications";
+import {Button, Dialog, DialogContent, DialogTitle, IconButton, styled} from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+import Settings from "../common/settings";
+import { useEffect, useState } from "react";
+import FetchHandler from "../common/FetchHandler";
+
+const GenerateDialogRoot = styled('div')({
+  width: '100%',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+
+  '> *': {
+    marginBottom: '16px',
+  }
+});
+
+const StyledButton = styled(Button)({
+    cursor: 'pointer',
+    width: '128px',
+    display: 'flex',
+
+    ':hover': {
+        backgroundColor: '#BFBFBF',
+    },
+});
+
+const StyledA = styled('a')({
+  textDecoration: "none",
+});
+
+const StyledIconButton = styled(IconButton)({
+    position: 'absolute',
+    top: 10,
+    right: 0,
+});
```

Comment:
```suggestion
const StyledDialogTitle = styled(DialogTitle)({
    display: "flex",
    justifyContent: "space-between"
});
```
Absolute positions should be avoided where possible because they just don't fit well in a dynamic layout.
This change has the same effect except for the abovementioned fit and retainment of the `DialogTitle`s padding (which absolute positioning just ignored)

Reply:
implemented in branch: "generatorpage-add-delete"